api/src/main/java/org/pmiops/workbench/conceptset/ConceptSetService.java

```diff
@@ -196,6 +196,7 @@ public ConceptSet getConceptSet(Long conceptSetId) {
   public List<ConceptSet> findByWorkspaceId(long workspaceId) {
     return conceptSetDao.findByWorkspaceId(workspaceId).stream()
         .map(conceptSetMapper::dbModelToClient)
+        .map(conceptSet -> toHydratedConcepts(conceptSet))
```

Comment:
nit: I think this can be done on one line.
```suggestion
        .map(conceptSet -> toHydratedConcepts(conceptSetMapper.dbModelToClient(conceptSet)))
```